src/ontology/mondo-edit.obo

```diff
@@ -421819,9 +421815,10 @@ xref: ICD10CM:E75.2 {source="Orphanet:447896", source="Orphanet:447896/attribute
 xref: MEDGEN:1842823 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: Orphanet:447896 {source="MONDO:equivalentTo"}
 xref: UMLS:C5680067 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:1842823"}
-is_a: MONDO:0004884 ! eye degenerative disorder
+is_a: MONDO:0004884 {source="Orphanet:447896"} ! eye degenerative disorder
+is_a: MONDO:0011897 {source="https://orcid.org/0000-0001-5493-2602"} ! leukodystrophy, hypomyelinating, 7, with or without oligodontia and/or hypogonadotropic hypogonadism
 is_a: MONDO:0100309 {source="Orphanet:447896"} ! hereditary ataxia
-relationship: excluded_subClassOf MONDO:0011897 {source="https://orcid.org/0000-0001-5208-3432"} ! leukoencephalopathy-ataxia-hypodontia-hypomyelination syndrome
+relationship: excluded_subClassOf MONDO:0011897 {source="https://orcid.org/0000-0001-5208-3432"} ! leukodystrophy, hypomyelinating, 7, with or without oligodontia and/or hypogonadotropic hypogonadism
```

Comment:
This SC is both added and excluded

Reply:
fixed!

I think I addressed everything now!